lib/entice/web/client.ex

```diff
@@ -85,11 +87,35 @@ defmodule Entice.Web.Client do
     {:ok, friends}
   end
 
+  @doc "Returns a friend's online status and character name from his account id."
+  def get_status(account_id) do
+    case Entice.Web.Repo.get(Entice.Web.Account, account_id) do
```

Comment:
i think before querying the db we can quickly shoot a query to the client server (below in this source file) - it would need a new dictionary for that purpose, since we now only order by emails, but that should be easy to do...